consensus-types/blocks/getters.go

```diff
@@ -666,6 +677,16 @@ func (b *SignedBeaconBlock) UnmarshalSSZ(buf []byte) error {
 				return err
 			}
 		}
+	case version.Gloas:
+		pb := &eth.SignedBeaconBlockGloas{}
+		if err := pb.UnmarshalSSZ(buf); err != nil {
+			return err
+		}
+		var err error
```

Comment:
I know this is the design on all previous cases, but is there any reason to not have done:

```suggestion
		err := pb.UnmarshalSSZ(buf); 
		if err != nil {
			return err
		}
```